src/test/java/APIBoundaryTest/ApplicationTest.java

```diff
@@ -1,7 +1,6 @@
 package APIBoundaryTest;
 
-import com.aviva.APIBoundary.Application;
-import org.junit.jupiter.api.BeforeEach;
```

Comment:
why did you remove the Beforeeach?

Reply:
It was actually gone, even before this PR so not sure why it's coming up in this PR. In that file, I don't do any repeated setup or anything because there is just one test so it was redundant.